plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsMysqlFlywayMigration.java

```diff
@@ -38,62 +36,4 @@ protected final boolean tableExists(String tableName)
                         .mapTo(Long.class)
                         .one()) > 0;
     }
-
-    @Test
-    public void testMigrationWithOldResourceGroupsSchema()
```

Comment:
I removed this test as it is not compatible with the new [constraint](https://github.com/trinodb/trino/pull/25113/files#diff-94c683474b608d9d93cf3dce00f0618044eab21187ce1725ebbf77a5126f8590). I feel this test is a bit flawed. 

I'm not sure why only this MySQL test supports old data during the migration. Postgres and Oracle tests do not have this test.